folio-spring-system-user/src/main/java/org/folio/spring/service/SystemUserService.java

```diff
@@ -81,6 +84,15 @@ public UserToken authSystemUser(SystemUser user) {
     return token;
   }
 
+  public UserToken authSystemUser(String tenantId, String username, String password) {
+    var token = getTokenWithExpiry(tenantId, username, password);
+    if (!isValidUserToken(token)) {
+      log.info("Login with expiry end-point returned null");
+      return getTokenLegacy(tenantId, username, password);
+    }
+    return token;
+  }
+
```

Comment:
All of this code already exists on the master branch... Why are we wanting to merge it again...

Reply:
I have updated the PR. please have a look.